lib/trento/operations/host_policy.ex

```diff
@@ -64,6 +71,39 @@ defmodule Trento.Operations.HostPolicy do
 
   defp get_saptune_operation_resource_id(_), do: nil
 
+  defp authorize_saptune_solution_operation(
```

Comment:
nitpick:
```
authorize_saptune_solution_operation(:saptune_solution_apply, nil), do: :ok
authorize_saptune_solution_operation(:saptune_solution_apply, %{applied_solution: nil}), do: :ok
authorize_saptune_solution_operation(:saptune_solution_apply, _), do: {:error,
         ["Cannot apply the requested solution because there is an already applied on this host"]}
authorize_saptune_solution_operation(:saptune_solution_change, %{applied_solution: applied_solution}) when not is_nil(applied_solution) -> :ok
authorize_saptune_solution_operation(:saptune_solution_change, _), do: {:error,
         [
           "Cannot change the requested solution because there is no currently applied one on this host"
         ]}
```

Reply:
thanks aab2e6405a0867f73ed5ca7323c1ba1d83e70eaa